modules/ping-protection/events/messageCreate.js

```diff
@@ -0,0 +1,113 @@
+const { 
+    addPing, 
+    getPingCountInWindow, 
+    executeAction, 
+    sendPingWarning 
+} = require('../ping-protection');
+const { localize } = require('../../../src/functions/localize');
+// Messages handler
+module.exports.run = async function (client, message) {
+    if (!client.botReadyAt) return;
+    if (!message.guild) return;
+    if (message.author.bot) return;
+    if (message.guild.id !== client.config.guildID) return;
+
+    const config = client.configurations['ping-protection']['configuration'];
+    const storageConfig = client.configurations['ping-protection']['storage'];
+    const moderationRules = client.configurations['ping-protection']['moderation'];
+    
+    if (!config || !moderationRules || !Array.isArray(moderationRules) || moderationRules.length === 0) return;
```

Comment:
```suggestion
    if (moderationRules.length === 0) return;
```
config and sotrageConfig values are ensured by core